test/node/extended_json.test.ts

```diff
@@ -181,6 +182,92 @@ describe('Extended JSON', function () {
     expect(EJSON.parse(serialized)).to.deep.equal(numbers);
   });
 
+  it('truncates bigint values when they are outside the range [BSON_INT64_MIN, BSON_INT64_MAX] in canonical mode', function () {
```

Comment:
Should I move these tests to live with the other BigInt support tests in `bigint.test.ts`?